clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -218,4 +243,66 @@ public boolean seekToNewSource(long l) throws IOException {
             return false;
         }
     }
+
+    /**
+     * Returns Location with repository, ref and path used by lakeFS based on filesystem path
+     * @param path
+     * @return
+     */
+    private Location pathToLocation(Path path) {
+        if (!path.isAbsolute()) {
+            path = new Path(this.workingDirectory, path);
+        }
+
+        URI uri = path.toUri();
+        if (uri.getScheme() != null && uri.getPath().isEmpty()) {
+            return null;
```

Comment:
Can you log a debug message saying why we are returning a null location?

Reply:
Here I don't think that logging is required - not an error and logging everything is too much.